meilisearch-core/Cargo.toml

```diff
@@ -11,6 +11,7 @@ bincode = "1.1.4"
 byteorder = "1.3.2"
 chrono = { version = "0.4.9", features = ["serde"] }
 compact_arena = "0.4.0"
+cow-utils = "0.1.2"
```

Comment:
Do we always need this dependency?

Reply:
No we indeed don't, since we now use unicase